src/main/resources/io/jenkins/plugins/designlibrary/Layouts/index.jelly

```diff
@@ -0,0 +1,144 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples" xmlns:st="jelly:stapler">
+  <s:sample>
+    <div class="jdl-component-sample">
+      <div id="layout-preview" class="jdl-layout-preview jdl-layout-preview--two-column">
+        <div class="jdl-layout-preview__navigation" />
+        <div class="jdl-layout-preview__sidebar">
+          <div class="jdl-layout-preview__header" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+        </div>
+        <div class="jdl-layout-preview__content">
+          <div class="jdl-layout-preview__header" />
+          <div class="jdl-layout-preview__button" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+        </div>
+        <div class="jdl-layout-preview__cards">
+          <div class="jdl-layout-preview__header" />
+          <div class="jdl-layout-preview__button" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+        </div>
+      </div>
+    </div>
+
+    <div class="jenkins-tab-pane">
+      <h2 class="jenkins-tab-pane__title">Two column</h2>
+
+      <p class="jdl-leading-paragraph">
+        Two column layouts are for 'hub' style pages, pages that have child pages
+        for the user to navigate to.
```

Comment:
I think (at least for post build actions) we have multi-column pages as well: one column for the navigation (or side panel) and several other columns for the actual content.

Reply:
Do you have a screenshot of that?

Reply:
- https://ci.jenkins.io/job/Plugins/job/analysis-model/job/master/1439/coverage/#overview
- https://ci.jenkins.io/job/Plugins/job/analysis-model/job/master/1439/spotbugs

Both use the BS5 grid of 12 columns

Reply:
I see, I'd still class those pages as being two column layouts though, with the content panel having sub-columns for the SpotBugs cards.

Reply:
Yes, that makes sense to me.

Maybe we can find a better/different naming? I'm not sure if I am the only one who is having the bootstrap grid for the content area in mind when we are talking about UI columns.